disnake/user.py

```diff
@@ -456,6 +468,75 @@ async def edit(
         return ClientUser(state=self._state, data=data)
 
 
+class Nameplate:
```

Comment:
This is missing a `__repr__`

Reply:
https://github.com/DisnakeDev/disnake/pull/1303/commits/2bb2b06b28c22b32c5e35df35ca96c08e8a70c5b